dir.props

```diff
@@ -13,6 +13,6 @@
     <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
 
 
-    <CliVersionPrefix>1.0.0-rc4</CliVersionPrefix>
+    <CliVersionPrefix>1.0.0</CliVersionPrefix>
```

Comment:
This got hoisted to the appropriate `--version-suffix` parameter in Compile.targets to work around NuGet/Home/#4404. We'll want to put this back once that is fixed so that the prefix can be stable along with the product's remaining branding.